cmd/release/cmd/generate.go

```diff
@@ -19,6 +19,12 @@ var (
 	k3sPrevMilestone string
 	k3sMilestone     string
 
+	uiPrevMilestone string
```

Comment:
These 2 values will be the same as the values below. We should store them once and reuse where/if necessary.

Reply:
Unified them as dashboard variables.